src/lib/core/CHIPError.h

```diff
@@ -602,7 +609,15 @@ using CHIP_ERROR = ::chip::ChipError;
  */
 #define CHIP_ERROR_INVALID_SIGNATURE                           CHIP_CORE_ERROR(0x14)
 
-// AVAILABLE: 0x15
+/**
+ *  @def CHIP_ERROR_INVALID_TLV_CHAR_STRING
+ *
+ *  @brief
+ *    Invalid TLV character string (e.g. null terminator)
+ *
+ */
+#define CHIP_ERROR_INVALID_TLV_CHAR_STRING                     CHIP_CORE_ERROR(0x15)
```

Comment:
Suggest "CHIP_ERROR_TLV_STRING_WITH_TRAILING_NULL" since it's the only case for now.

Reply:
Hope is that spec will actually pick up https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/8022 and then it will not only be trailing nulls.

I am thinking to keep this as  "this was bad" and not try to be as specific so I do not need to add even more codes later.

Reply:
Discussed offline, lets keep it somewhat vague to be able to do either 'null at end' or 'null somewhere' depending on where spec leads us to in time.